sr/comp/cli/show_schedule.py

```diff
@@ -30,6 +30,18 @@ def command(settings):
 
         matches = matches[:int(settings.limit)]
 
+    if settings.team:
+        if settings.team not in comp.teams.keys():
+            print('TLA not found')
+            return
```

Comment:
Since this is an error case, please could we:
- include the "bad" data in the message,
- put the error message on `stderr`, and
- return an error code so that (scripted) callers know something went wrong

I think we have some handlers for this sort of thing in the deploy script if you want some inspiration.
Aside: I've long been meaning to formalise this sort of thing though, likely styled after Django's `CommandError` exception, though that's likely overkill to introduce in this PR.

One approach here might be:
```suggestion
            exit(f"TLA {settings.team!r} not found")
```